flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/CheckpointExceptionHandlerConfigurationTest.java

```diff
@@ -63,6 +63,10 @@ private void testConfigForwarding(boolean failOnException) throws Exception {
 		environment.setTaskStateManager(new TestTaskStateManager());
 		environment.getExecutionConfig().setFailTaskOnCheckpointError(expectedHandlerFlag);
 
+		if (failOnException) {
+			environment.getExecutionConfig().setTaskTolerableCheckpointFailuresNumber(3);
```

Comment:
this can be always set, as it should be just ignored if `failOnException` is `false`